packages/app/src/debug/DebugErrored.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <Alert
+    :title="t('debugPage.incomplete')"
+    status="warning"
+    :icon="ErrorOutlineIcon"
+    class="flex flex-col mb-24px w-full"
+  >
+    <div class="ml-5px">
+      <ul
```

Comment:
I'm seeing the list item bullet cut off on the left side of the alert box when I run the tests for this component locally

Reply:
Which test are you seeing this in? I haven't been able to see it

Reply:
More borked local state, sorry